src/app/personside/kontrollsporsmal/skjulPåTversAvVinduerUtils.ts

```diff
@@ -1,19 +1,36 @@
 import moment from 'moment';
 import * as Cookies from 'js-cookie';
 
-const cookieNavn = 'SkjulKontrollSpørsmålFor';
+const skjulForCookieNavn = 'SkjulKontrollSpørsmålFor';
```

Comment:
Tror ikke helt jeg forstår dette navnet. Blir det lest: "skjul kontrollspørsmål for Z991742"?, men det ligger på den samme brukeren sin browser? Blir det ikke mer intuitivt med en slags boolean cookie?